### ISSUE TITLE
Problem with MissingTemplate and ignored fiels

### ISSUE THREAD

**Issue by MBO** [Shopify/theme-check#489]
We've found the issue while using 3rd party Shopify apps (Langify for translation and Wisepops).

Those apps add snippet files to the repository, but because the files are 3rd party and could be updated any time, we don't want to refactor them. Once we add those files to `.theme-check.yml` to `ignore` section theme-check's MissingTemplate starts to warn that those files are missing, but they are on the disc, they are in the repository and they are uploaded to shopify theme store when we deploy, it's just we don't want to validate those because it's not ours code.

**Comment by MBO** [comment]
I checked your solution and it works, but now I have to put mu ignored snippet filename twice:

```
ignore:
  - snippets/a.liquid
  - snippets/b.liquid
  - snippets/c.liquid
  - snippets/d.liquid
  - snippets/e.liquid
  - sections/f.liquid
  - 'layout/theme_backup_*'
  - templates/page.gem-*
  - templates/collection.gem-*
  - assets/gem-page-*

MissingTemplate:
  ignore_missing:
    - snippets/a.liquid
    - snippets/b.liquid
    - snippets/c.liquid
    - snippets/d.liquid
    - snippets/e.liquid
    - sections/f.liquid
```

So I have to copy snippet/section names, and only backup and template files are safe to ignore globally only once.

But this solves the issue only partially - we don't get a warning. But it won't save us when for some reason we don't have the file in the repository, i.e. it was not committed by a developer, or removed but the reference was left in the code.

It would be nice to have validated that the file is in the directory, just not lint its content.

**Comment by charlespwd** [comment]
Actually, I don't think I can solve the "I want to make sure it exists, but not check its content problem" for performance reasons in the language server (we need an InMemoryStorage abstraction for buffers and stuff, and wouldn't want to load all node_modules in there because of the MissingSnippets check).

#613 will make it so you will need to only ignore it once, however. Updated config that will do the same:

```yaml
ignore:
  - snippets/a.liquid
  - snippets/b.liquid
  - snippets/c.liquid
  - snippets/d.liquid
  - snippets/e.liquid
  - sections/f.liquid
  - 'layout/theme_backup_*'
  - templates/page.gem-*
  - templates/collection.gem-*
  - assets/gem-page-*

MissingTemplate:
  enabled: true
```

You won't need to ignore_missing if it's ignored is what I'm saying.
